app/api/recommend.py

```
@@ -183,12 +198,15 @@ def hybrid_recommend(
         }
         recommendation_items.append(item)
     final_results = {"recommendationsList": recommendation_items}
+    logger.debug(f"[RECO][DONE] endpoint={endpoint} userId={user_id} result={final_results}")
 
     # 6. 캐시 저장 (20분)
     try:
         r.setex(cache_key, 1200, json.dumps(final_results))
+        latency_ms = int((time.perf_counter() - t0) * 1000)
+        logger.info(f"[CACHE][SET] ttl={ttl_sec} latencyMs={latency_ms} key={cache_key_masked}")
```

Ruff (0.12.2)

Undefined name `ttl_sec`

(F821)